Neos.Flow/Classes/ObjectManagement/Proxy/Compiler.php

```diff
@@ -199,7 +199,16 @@ public function compile()
                 }
             }
         }
-        return $classCount;
+        $this->emitCompiledClasses($compiledClasses);
```

Comment:
My first approach had a signal for each single class that would receive the `ProxyClass` as second argument. That might be useful for enhancing the proxies with hooked in functionality via signal/slot, but it would mean that the CacheManager slot would have to somehow receive another signal when all classes are finished or keep reflushing caches again and again.